code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3917,6 +3917,26 @@
 	required_reagents = list(PICCOLYN = 1, INACUSIATE = 1, SUGAR = 1)
 	result_amount = 3
 
+/datum/chemical_reaction/changeling_blood/New()
+    ..()
+	//welders are limited to ~3500K.
+	//roughly 50% chance of failing when using welder
+    required_temp += 3300 + 3.3*rand(-1000, 1000)
```

Comment:
This is super janky. Basically at roundstart it will pick an amount of temperature you need to raise ling blood to for a successful test. That remains the same the whole round, so sometimes ALL lings have blood that can be angered with a welder, or all lings are above that threshold.